server/server.js

```diff
@@ -25,510 +27,483 @@ const unrespondedSessionAlertTimeoutMillis = helpers.getEnvVar('FALLBACK_TIMEOUT
 app.use((req, res, next) => {
-    if (req.cookies.user_sid && !req.session.user) {
-        res.clearCookie('user_sid');
-    }
-    next();
-});
-
+  if (req.cookies.user_sid && !req.session.user) {
+    res.clearCookie('user_sid')
+  }
+  next()
+})
 
 // middleware function to check for logged-in users
-var sessionChecker = (req, res, next) => {
-    if (req.session.user && req.cookies.user_sid) {
-        res.redirect('/dashboard');
-    }
-    else {
-        next();
-    }
-};
-
+function sessionChecker(req, res, next) {
+  if (req.session.user && req.cookies.user_sid) {
+    res.redirect('/dashboard')
+  } else {
+    next()
+  }
+}
 
 app.get('/', sessionChecker, (req, res) => {
-    res.redirect('/login');
-});
+  res.redirect('/login')
+})
 
+app
+  .route('/login')
+  .get(sessionChecker, (req, res) => {
+    res.sendFile(`${__dirname}/login.html`)
+  })
+  .post((req, res) => {
+    const username = req.body.username
+    const password = req.body.password
 
-app.route('/login')
-    .get(sessionChecker, (req, res) => {
-        res.sendFile(__dirname + '/login.html');
-    })
-    .post((req, res) => {
-        var username = req.body.username,
-            password = req.body.password;
-
-        if ((username === helpers.getEnvVar('WEB_USERNAME')) && (password === helpers.getEnvVar('PASSWORD'))) {
-            req.session.user = username;
-            res.redirect('/dashboard');
-        } 
-        else {
-            res.redirect('/login');
-        }
-    });
+    if (username === helpers.getEnvVar('WEB_USERNAME') && password === helpers.getEnvVar('PASSWORD')) {
+      req.session.user = username
+      res.redirect('/dashboard')
+    } else {
+      res.redirect('/login')
+    }
+  })
 
 app.get('/dashboard', async (req, res) => {
-    if (!req.session.user || !req.cookies.user_sid) {
-        res.redirect('/login')
-        return
-    }
-          
-    try {
-        let allInstallations = await db.getInstallations()
-                
-        let viewParams = {
-            installations: allInstallations
-                .filter((installation) => installation.isActive)
-                .map((installation) => {
-                    return { name: installation.name, id: installation.id }
-                }),
-        }
-        viewParams.viewMessage = allInstallations.length >= 1 ? 'Please select an installation' : 'No installations to display'
-          
-        res.send(Mustache.render(chatbotDashboardTemplate, viewParams))
-    } catch (err) {
-        helpers.log(err)
-        res.status(500).send()
-    }
+  if (!req.session.user || !req.cookies.user_sid) {
+    res.redirect('/login')
+    return
+  }
+
+  try {
+    const allInstallations = await db.getInstallations()
+
+    const viewParams = {
+      installations: allInstallations
+        .filter(installation => installation.isActive)
+        .map(installation => ({ name: installation.name, id: installation.id })),
```

Comment:
And this one, I don't understand why it needs the brackets. Could you just have
`.map(installation => { name: installation.name, id: installation.id }),`?

Reply:
I'm not sure, this is one of the auto ones, I can check

Reply:
If I do that it says that name & id are unused labels and that : is an unexpected token, I think because return is gone? And when I put return back it says (after making me restyle it lol) "Unexpected block statement surrounding arrow body;  parenthesise the returned value and move it immediately after the `=>`. eslint(arrow-body-style)"

Reply:
Cool. Today I learned :-)